libnmstate/state.py

```diff
@@ -264,3 +269,41 @@ def dict_update(origin_data, to_merge_data):
         else:
             origin_data[key] = val
     return origin_data
+
+
+class LinuxBridgeState(object):
+    DEFAULT_STP_ENABLED = True
+    DEFAULT_PORT_STP_HAIRPIN_MODE = False
+    DEFAULT_PORT_STP_PATH_COST = 100
+    DEFAULT_PORT_STP_PRIORITY = 32
+
+    DEFAULT_PORT_STP_OPTIONS = {
+        LinuxBridge.PORT_STP_HAIRPIN_MODE: DEFAULT_PORT_STP_HAIRPIN_MODE,
+        LinuxBridge.PORT_STP_PATH_COST: DEFAULT_PORT_STP_PATH_COST,
+        LinuxBridge.PORT_STP_PRIORITY: DEFAULT_PORT_STP_PRIORITY,
+    }
+
+    STP_OPTIONS = [
```

Comment:
It might be a good idea to use a tuple instead of a list here, since it is immutable.